platform/Tcp.roc

```diff
@@ -112,9 +112,10 @@ connect = \host, port ->
         |> Result.mapErr parseConnectErr
     |> InternalTask.fromEffect
 
-close : Stream -> Task {} *
+close : Stream -> Task {} []
 close = \@Stream stream ->
     Effect.tcpClose stream
+    |> Effect.map Ok
```

Comment:
`tcpClose` previously returned a `Result`, now it returns `{}`, so we can use `Effect.map Ok`.